mozilla/fx-private-relay
# create Google app creds file from env var as needed

This PR fixes MPP-3467.

It changes the profiling code to automatically create the necessary GCP key JSON file from the value of the `GOOGLE_CLOUD_PROFILER_CREDENTIALS_B64` environment variable. This should work identically across local, dev, stage, and prod environments.

How to test:
1. Follow [the doc](https://github.com/mozilla/fx-private-relay/pull/4366/files#diff-01748e8486974d80818eb9b8c9f623769f0ddb0821f867644d6e113f0dc008d3)!
2. When you `runserver` you should see output lines: `HTTPAdapter.get_connection url: http://metadata/computeMetadata/v1/instance/zone` which indicates that the profiler code is working
